sw/host/opentitanlib/src/backend/chip_whisperer.rs

```diff
@@ -15,6 +15,11 @@ pub struct ChipWhispererOpts {
     /// Comma-separated list of Chip Whisperer board UARTs for non-udev environments. List the console uart first.
     #[arg(long, alias = "cw310-uarts")]
     pub uarts: Option<String>,
+
+    /// Path to OpenOCD JTAG adapter config file to use (usually Olimex, but could be another
+    /// adapter).
+    #[arg(long)]
+    pub openocd_adapter_config: Option<String>,
```

Comment:
I think it should probably be a `Option<PathBuf>`, I am not sure why we defined the paths as `Option<String>` when we did the openocd backend. Can you try and see if that causes any issue?

Reply:
I have changed the parameter into `Option<PathBuf>`.  I have also added a small refactor CL to this PR, which eliminates the use of  `Vec<&str>` in the starting of the OpenOCD sub-process, in order to allow some of the arguments to be `&PathBuf` while others remain `&str`.